### ISSUE TITLE
warning message! `funs()` was deprecated in dplyr 0.8.0.

### ISSUE THREAD

**Issue by adriantich** [tobiasgf/lulu#18]
Hello, I have the following warning. It is not FATAL yet but could be in a future I guess!

**Warning message:
`funs()` was deprecated in dplyr 0.8.0.
ℹ Please use a list of either functions or lambdas:

# Simple named list: list(mean = mean, median = median)

# Auto named with `tibble::lst()`: tibble::lst(mean, median)

# Using lambdas list(~ mean(., trim = .2), ~ median(., na.rm = TRUE))
ℹ The deprecated feature was likely used in the lulu package.
  Please report the issue to the authors.** 

Cheers!
Adrià

**Comment by adriantich** [comment]
Sorry for the font format, it was not intended! :(

**Comment by slambrechts** [comment]
I see the same warning, do you know if this means lulu did not run correctly?

**Comment by adriantich** [comment]
I don't think so, as long as it is only a warning there is no problem. But that means that in future versions this will rise an error.

**Comment by gabrielet** [comment]
i am contacting the maintainer of the `lulu` package to let them know that it's an easy fix. let's see!
